package.json

```diff
@@ -23,8 +23,9 @@
     "clean:build": "rimraf build && rimraf build2",
     "dev": "webpack --config webpack.dev.config.js",
     "lint": "eslint .",
+    "tsd": "scripts/tsd.sh",
```

Comment:
Can we config tsd to point it to the test/types folder? So we don't need the script anymore.